src/main/java/de/tum/cit/aet/artemis/athena/config/AthenaHealthIndicator.java

```diff
@@ -91,17 +79,14 @@ public Health health() {
     /**
      * Get additional information about the health of the assessment modules.
      */
-    private Map<String, Object> getAdditionalInfoForModules(JsonNode athenaHealthResponse) {
+    private Map<String, Object> getAdditionalInfoForModules(AthenaHealthResponse athenaHealthResponse) {
         var additionalModuleInfo = new HashMap<String, Object>();
-        if (athenaHealthResponse.has(ATHENA_MODULES_KEY)) {
-            JsonNode modules = athenaHealthResponse.get(ATHENA_MODULES_KEY);
-            // keys are module names, values are description maps
-            modules.fields().forEachRemaining(module -> {
-                var moduleHealth = new AthenaModuleHealth(module.getValue().get(ATHENA_MODULE_EXERCISE_TYPE_KEY).asText(),
-                        module.getValue().get(ATHENA_MODULE_HEALTHY_KEY).asBoolean(), module.getValue().get(ATHENA_MODULE_URL_KEY).asText());
-                additionalModuleInfo.put(module.getKey(), moduleHealthToString(moduleHealth));
-            });
-        }
+        var modules = athenaHealthResponse.modules();
+        // keys are module names, values are description maps
+        modules.forEach((key, value) -> {
+            var moduleHealth = new AthenaModuleHealth(value.type(), value.healthy(), value.url());
+            additionalModuleInfo.put(key, moduleHealthToString(moduleHealth));
+        });
```

Comment:
I think that we would not need the comment if we renamed the key and value directly 
```suggestion
        modules.forEach((moduleName, descriptionMap) -> {
            var moduleHealth = new AthenaModuleHealth(descriptionMap.type(), descriptionMap.healthy(), descriptionMap.url());
            additionalModuleInfo.put(moduleName, moduleHealthToString(moduleHealth));
        });
```